addons/pos_restaurant/static/src/js/floors.js

```diff
@@ -77,6 +77,9 @@ models.Order = models.Order.extend({
         json.customer_count = this.get_customer_count();
         return json;
     },
+    _should_be_reset: function() {
+        return _super_order._should_be_reset.apply(this, arguments) && !this.pos.config.module_pos_restaurant;
```

Comment:
I am not sure it should depend if the pos_restaurant module is installed or not, but if we are on a bill or a receipt

Reply:
Using `iface_printbill` ?
https://github.com/odoo/odoo/blob/379e4f0a72bc95798068735c0c7a8662d3a48b3b/addons/pos_restaurant/models/pos_config.py#L11
I saw the condition to display the "Bill" button uses the variable:
https://github.com/odoo/odoo/blob/d8726778b670611326851d9baa980edbd21b3b09/addons/pos_restaurant/static/src/js/Screens/ProductScreen/ControlButtons/PrintBillButton.js#L28-L33